proto/settlus/oracle/v1alpha1/oracle.proto

```diff
@@ -14,6 +14,7 @@ message Params {
     (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
     (gogoproto.nullable) = false
   ];
+  // DEPRECATED: tolerated_error_band have been deprecated
```

Comment:
why did it get deprecated?

Reply:
In the case where consensus had been halted for a long time and has only recently been achieved again, as we experienced with the devnet. Imo, since everything is determined by consensus, there is no real need for stricter control.

Reply:
`tolerated_error_band` does not halt consensus. (my bad on wrongly explaining what it does 🥲)
https://github.com/settlus/chain/blob/318f4d9700db5b39e7cbd1df5139a35df213ec31/x/oracle/abci.go#L109

it helps validators not get punished for submitting height that is close to the consensus. here is an example:
```yaml
correct_height: 100
tolerated_error_band: 3

validator_foo_answer: 100 # correct
validator_bar_answer: 102 # correct because abs(102 - 100) = 2 is less than the tolerated error band of 3
validator_baz_answer: 95 <- wrong because abs(95-100) - 5 is larger than the tolerated error band of 5
```
to summarize, `tolerated_error_band` is only about rewarding validator which does not halt consensus.

Reply:
It shouldn't be happened after the `timestamp` is added.